extensions/wikia/SpitfiresContributionExperiments/scripts/my-profile-tour.js

```diff
@@ -0,0 +1,177 @@
+require(['jquery', 'ext.wikia.spitfires.experiments.tracker', 'wikia.loader', 'wikia.nirvana', 'wikia.mustache', 'mw'],
+	function ($, tracker, loader, nirvana, mustache, mw) {
+		'use strict';
+
+		var experimentName = 'CONTRIB_EXPERIMENTS',
+			modalConfig = {
+				vars: {
+					id: 'MyProfileModal',
+					classes: ['my-profile-modal'],
+					size: 'medium', // size of the modal
+					content: '' // content
+				},
+				confirmCloseModal: function () {
+					if (!saved && Object.keys(answers).length) {
+						saveProfile();
+					} else {
+						if (saved) {
+							window.location.href = wgServer + '/wiki/' + userPage;
+						}
+						return true;
+					}
```

Comment:
I think it would look cleaner with an explicit `return false;` for this function to make it clear it doesn't close the modal.